src/Symfony/Installer/DownloadCommand.php

```diff
@@ -89,9 +88,12 @@ protected function download()
 
         /** @var ProgressBar|null $progressBar */
         $progressBar = null;
-        $downloadCallback = function ($size, $downloaded, $client, $request, Response $response) use (&$progressBar) {
-            // Don't initialize the progress bar for redirects as the size is much smaller
-            if ($response->getStatusCode() >= 300) {
+        $downloadCallback = function (ProgressEvent $event) use (&$progressBar) {
+            $size = $event->downloadSize;
+            $downloaded = $event->downloaded;
+
+            if ($size < 1 * 1024 * 1024) {
```

Comment:
I guess not :smile: It's just a personal preference to read it more clearly as "1 megabyte". Should I change it?